packages/cli/src/commands/decode.ts

```diff
@@ -75,7 +84,7 @@ export async function decode({
       log(errorMessage);
       return;
     }
-    throw new Error('Could not decode transaction data');
+    throw new Error('Could not decode transaction data with the given Cannon package');
```

Comment:
add some advice like "Please confirm that the given data exists in the ABIs of the given package."